src/ai/backend/client/func/quota_scope.py

```diff
@@ -0,0 +1,157 @@
+import textwrap
+from typing import Any, Sequence
+
+from ...common.types import QuotaScopeID
+from ..output.fields import group_fields, quota_scope_fields, user_fields
+from ..output.types import FieldSpec
+from ..session import api_session
+from .base import BaseFunction, api_function
+
+_default_user_fields = (
+    user_fields["uuid"],
+    user_fields["username"],
+)
+
+_default_project_fields = (
+    group_fields["id"],
+    group_fields["name"],
+)
+
+_default_detail_fields = (
+    quota_scope_fields["usage_bytes"],
+    quota_scope_fields["usage_count"],
+    quota_scope_fields["hard_limit_bytes"],
+)
+
+_default_quota_scope_fields = (
+    quota_scope_fields["quota_scope_id"],
+    quota_scope_fields["storage_host_name"],
+)
+
+
+class QuotaScope(BaseFunction):
+    @api_function
+    @classmethod
+    async def get_user_info(
+        cls,
+        domain_name: str,
+        email: str,
+        fields: Sequence[FieldSpec] = _default_user_fields,
+    ):
+        query = textwrap.dedent(
+            """\
+            query($domain_name: String!, $email: String!) {
+                user(domain_name: $domain_name, email: $email) {$fields}
+            }
+        """
```

Comment:
`domain_name` input is not required for querying individual `user` object. Requiring corresponding domain name to every query to CLI user can be cumbersome. Let's just remove it.